src/docs/ContributionGuide.md

```diff
@@ -1,38 +1,55 @@
-# How-to "knowledge base"
+---
+layout: docs
```

Comment:
I must say i do not really agree with this decision. If the change is more than just fixing a typo the change author should actually use an markdown viewer or be knowledgeable enough about markdown to use the README.md, which also has the added advantage to be displayed in GitHub if you navigate to the [docs](https://github.com/nfdi4plants/nfdi4plants.knowledgebase/tree/main/src/docs) folder and to keep and unified naming.

In addition i would advice against using README.md and ContributionGuide.md, as this would create maintenance issues.

**What we CAN do is:**
Start parsing README.md, so we also have an html. In this case we would need to add the docs layout metadata to the README.md but otherwise it would be the same.

Reply:
> Start parsing README.md, so we also have an html. In this case we would need to add the docs layout metadata to the README.md but otherwise it would be the same.

Yes! Let's do that.

Reply:
For ignored stuff we have the _ignored folder. Every other md should be rendered (and then requires yaml)

Reply:
Will need to implement the README.md parsing, but thats no issue. Just so you know it won't be available out of the box.